crime-means-assessment/src/test/java/uk/gov/justice/laa/crime/meansassessment/util/RoundingUtilsTest.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.justice.laa.crime.meansassessment.util;
+
+import org.junit.Test;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Modifier;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class RoundingUtilsTest {
+    @Test
+    public void testRoundingUtilConstructorIsPrivate() throws NoSuchMethodException {
+        assertEquals("There must be only one constructor", 1,
```

Comment:
Can we make sure we're using AssertJ fluent asserts here e.g. assertThat().isEqualTo(). The majority of our other test classes do this and we should be aiming to maintain consistency.